stable/postgresql/values.yaml

```diff
@@ -0,0 +1,289 @@
+## Global Docker image registry
```

Comment:
Default correct production values

Reply:
do you want to swap `values` <-> `values-production` ?